app/views/unattended/provisioning_templates/host_init_config/host_init_config_default.erb

```diff
@@ -54,6 +54,11 @@ trap 'exit_and_cancel_build' ERR
 
 <% end -%>
 
+<% if plugin_present?('katello') && @host.operatingsystem.family == 'Redhat' -%>
+echo "Refreshing subscription data"
+subscription-manager refresh
+<% end -%>
+
```

Comment:
I'd still prefer not to add an empty line in the rendered template unconditionally:
```suggestion

<% end -%>
```